lib/pg_repack.sql.in

```diff
@@ -38,21 +40,25 @@ AS $$
             from '\d+\.\d+(?:\.\d+)?'), '.')::int[]
     ) as x (tokens);
 $$
-LANGUAGE SQL IMMUTABLE;
+LANGUAGE SQL IMMUTABLE
+SET search_path to 'pg_catalog';
+
 
 CREATE AGGREGATE repack.array_accum (
-    sfunc = array_append,
-    basetype = anyelement,
-    stype = anyarray,
+    sfunc = pg_catalog.array_append,
+    basetype = pg_catalog.anyelement,
+    stype = pg_catalog.anyarray,
```

Comment:
The extension is in fact completely non-relocatable (type 3 in https://www.postgresql.org/docs/current/static/extend-extensions.html#EXTEND-EXTENSIONS-RELOCATION). `CREATE EXTENSION pg_repack SCHEMA foo` does nothing special apart from injecting a vulnerability.

I would add `schema = repack` into `pg_repack.control.in`. It will effectively make this whole file executed with `set search_path = 'repack'`, so we won't need to secure anything further to make _execution_ of this file safe. This would fix the installation step only.